modules/ROOT/pages/mule-app-properties-to-configure.adoc

```diff
@@ -1,37 +1,80 @@
-= Configure Property Placeholders for Mule Apps
+= Configure Property Placeholders
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Instead of using static values for connections and other configurations in your Mule app, you can use the procedure described here to create and use property placeholders that you define in a `.yaml` or `.properties` configuration file.
+Instead of using static values for connections and other configurations in your Mule application, you can create a `.yaml` or a `.properties` file to contain your properties and then reference the properties from your application.
+Configuring a properties file improves the organization and maintainability of your applications.
 
-In a YAML file, the properties take this form:
+In a `.yaml` file, the properties take this form:
+[source, yaml]
 ----
 http:
-  path: service
+  path: "/service"
   port: "10000"
+  host: "my-api.cloudhub.io"
 ----
 
 In a `.properties` file, the properties take this form:
 ----
-http.path=service
-http.port="10000"
+http.path=/service
+http.port=10000
+http.host=my-api.cloudhub.io
 ----
 
-You must store the file in the project directory for your Mule app:
+== Create and Configure your Properties File
 
-* `/src/main/resources`
+. Create a new `.yaml` or `.properties` file in your project, and store it in `/src/main/resources`.
+. Edit the file to define the properties and values you need.
+. Add the properties file to your Mule application.
```

Comment:
Adding the properties file isn't it the same as in step 1?
> Create a new `.yaml` or `.properties` file in your project, and store it in `/src/main/resources`.

Would the third step be something like:
> Use the values defined on your properties file as configurations for your Mule application
Or something like that.

Reply:
The first step creates the file, the third step links that file to the mule application. 
If the third step is not done, the properties cannot be referenced. Then, comes the next section that explains how to use the values from this file.